start.sh

```diff
@@ -81,11 +81,14 @@ if [ "${IS_CRON_POD:-"false",,}" == "false" ]; then
         echo "Starting Gunicorn for PTVSD debugging"
         # Workers need to be set to 1 for PTVSD
         GUNICORN_WORKERS=1
+        GUNICORN_RELOAD="--reload"
```

Comment:
The reload is happening when `"PTVSD_ENABLE": true,` but not every-time the python code changes. 

```
student_dashboard  | [2020-12-02 12:24:42 -0500] [32] [INFO] Worker reloading: /code/dashboard/views.py modified
student_dashboard  | [2020-12-02 12:24:42 -0500] [32] [INFO] Worker exiting (pid: 32)
student_dashboard  | [2020-12-02 12:24:43 -0500] [44] [INFO] Booting worker with pid: 44

```

Reply:
Are you saying there's some problem with Gunicorn reloading changes? Or that this is working as expected?

Reply:
I can see gunicorn server is starting up with out any issue when extra setting are removed from the `.env` and `docker-compose`.

But I don't see reload is happening as frequently I need when python code change. Infact I only saw one time. I don't know this is something specific to my machine

Reply:
Also I have noticed when running with "PTVSD_ENABLE": true ( i.e one worker ) things are really slow. the website initial load happens and when I reload things are very slow. I tested this from standalone tool.

Reply:
Yeah, it will be a lot slower. We won't be running in production on this mode though. But this should have been about the same performance as before as runserver is single threaded. 

Can't use ptvsd in a multithreaded environment. Not sure if debugpy changes that.

These things you're mentioning related to code changes or performance aren't related to this change though. Can open a new issue and look at that more if you wanted but this really should get in otherwise it will cause issues with production instances.

Reply:
Agreed. But I feel Python code changes are not picked is concerning for me when developing. Might be that it is purely my issue if other are not seeing it. I haven't being working on python code lately so i did not notice this earlier

Reply:
I am fine with slowness when running with ptvsd enabled. I will enable it when need to use the breakpoint feature